Improve readme: "run webpack"

The readme says

> Run webpack: 
$ webpack

- [ ] it should be stated as a dependency `npm install --user webpack`
- [ ] how to run it? There's no command `webpack`

@rhyolight I installed webpack:
`sudo npm install --global webpack`
(had to do the global, as the startup script had a problem with local)
now `webpack` command exists on my system

`npm start` still fails with

```
ell-viz$ npm start

> cell-viz@1.0.0 start /mnt/store/devel/HTM/htm_schools/cell-viz
> webpack-dev-server

module.js:328
    throw err;
    ^

Error: Cannot find module 'webpack'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/mnt/store/devel/HTM/htm_schools/cell-viz/node_modules/webpack-dev-server/bin/webpack-dev-server.js:17:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)

npm ERR! Linux 4.10.0-20-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "start"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! cell-viz@1.0.0 start: `webpack-dev-server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cell-viz@1.0.0 start script 'webpack-dev-server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the cell-viz package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack-dev-server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs cell-viz
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls cell-viz
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /mnt/store/devel/HTM/htm_schools/cell-viz/npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -1,7 +1,16 @@
+# Cell-viz
+
 This is an attempt at recreating an old illustration.  See it live at
 https://numenta.github.io/cell-viz/ or fiddle with it at http://jsfiddle.net/7tbm3mv1/4/.
 
-Usage
+## Dependencies
+
+```
+npm install webpack
+```
+Webpack can be installed and run [locally](https://webpack.js.org/guides/installation/#local-installation), or [globally](https://webpack.js.org/guides/installation/#global-installation)
+
+## Usage
 -----
 
 Run a simple server suitable for development:
@@ -19,7 +28,7 @@ $ webpack
 
 Use `out/dyson.js`;
 
-Examples
+## Examples
 --------
 
 Open `example1.html`, `example2.html`, or `example3.html`.

